### DISCUSSION TITLE
why '...' is rendered as ' ?.' ?

### DISCUSSION THREAD

**Discussion by zhang-stephen** [ryanoasis/nerd-fonts#799] [Q&A]
as title said, the `...` is rendered as ` ?.`. Is it a bug? Or how to disable it?

Font:
 + `FuraCode NF` with "windows compatible"

Terminal:
 + Windows Terminal On Windows 10 20H2

Others:
 + bash 4.2 on CentOS 7
 + neovim 0.6.1

**Comment by Finii** [comment]
Moved to Issue #800
